stefaniaPopescu26/assignment3/WordSearch.java

```diff
@@ -0,0 +1,148 @@
+import javafx.util.Pair;
```

Comment:
Why did you decide to use javafx? I don't think this is a very good idea. This is not part of open JDK and deciding to use it limits your project's portability. Also adding a new dependency to a project just for a tiny utility class like that seems too much.
Maybe just define an inner class Pair that holds two integers? That should be enough for your needs here